earthpy/spatial.py

```diff
@@ -229,10 +228,23 @@ def _stack_bands(sources, dest=None, write_raster=False):
     Parameters
     ----------
     sources : list of rasterio dataset objects
-        A list with paths to the bands you wish to stack. Objects
+        A list of rasterio dataset objects you wish to stack. Objects
         will be stacked in the order provided in this list.
-    dest : a rio.open writable object that will store raster data.
-    write_raster : a flag to decide to write out the raster.
+    dest : string (optional)
+        Path to the where the output raster containing the stacked
+        layers will be stored.
+    write_raster : bool (optional)
+        Boolean to determine whether or not to write out the raster.
+
+    Returns
+    ----------
+    tuple
+
+        numpy array
+            Numpy array generated from the stacked array combining all
+            bands that were provided in the list.
+        ret_prof : rasterio profile
+            Updated rasterio profile with the correct number of bands.
```

Comment:
How about:
```
ret_prof : rasterio profile object
            Updated rasterio spatial metadata object updated to represent the number of layers in the stack.
```

Reply:
looks like a few comments were missed! i'm adding a few commits so we can merge this @nkorinek